tests/onnxifi/build_onnxifi_tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Should be run from Glow source directory.
```

Comment:
Btw, one trick I use with this type of utility script is to get the script dir and then find the repo root from there, to avoid needing to run the script from a specific directory:
```
SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
GLOW_DIR="$SCRIPT_DIR/../.."
```